blockchain/substrate.go

```diff
@@ -191,12 +202,16 @@ func (sm *substrateManager) ParseResponse(data []byte) ([]subscriber.Event, bool
 	var subEvents []subscriber.Event
 	for _, change := range changes.Changes {
 		if !types.Eq(change.StorageKey, sm.key) || !change.HasStorageData {
+			logger.Error("Does not match storage")
 			continue
 		}
 
 		events := EventRecords{}
 		err = types.EventRecordsRaw(change.StorageData).DecodeEventRecords(sm.meta, &events)
 		if err != nil {
+			logger.Info("change.StorageData:", change.StorageData)
+			logger.Info("sm.key:", sm.key)
+			logger.Info("types.EventRecordsRaw:", types.EventRecordsRaw(change.StorageData))
```

Comment:
Does it make sense to make these log as debug?

You should also be able to do something like this: `logger.Debugw("Failed parsing", "change.StorageData", change.StorageData, "sm.key", sm.key [...])`

Reply:
My point here was to have the log without having to change the log level (the next line is a  `logger.error`).
I will change it to the  `logger.Debugw` :)

Reply:
Ah yeah actually you can probably do all on one line as `logger.Errorw()` then!

Reply:
Sure, but it will defeat the purpose of your first comment no ? I mean, having not to many logs ?

Reply:
The purpose was to have the correct log-level and combining them together in the same message. I think having them all in `logger.Errorw()` will give the best combination of context for the error and ease of extracting the logs.

Reply:
Ok great :+1: